Formula/f/fabric-ai.rb

```diff
@@ -0,0 +1,21 @@
+class FabricAi < Formula
+  desc "Open-source framework for augmenting humans using AI"
+  homepage "https://danielmiessler.com/p/fabric-origin-story"
+  url "https://github.com/danielmiessler/fabric/archive/refs/tags/v1.4.156.tar.gz"
+  sha256 "1b774702dd4e4933411379a65ebd314c84e592b7265ab1ca08e846b8271739f3"
+  license "MIT"
+  head "https://github.com/danielmiessler/fabric.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
```

Comment:
this way, the binary artifact would align with the upstream docs
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"fabric")
```

Reply:
@chenrui333 `fabric-ai` is used because it conflicts with another popular tool `fabric` that's [already in Homebrew](https://formulae.brew.sh/formula/fabric) and everywhere across different distributions. I'll revert it along with the recent version update.

I already opened an issue to update documentation in the future. Because the original `fabric` tool has been here for over a decade and is not going anywhere.

Reply:
yeah, this wont change the formula name (and user would still do `brew install fabric-ai`), but just the binary artifact, and we have a way to avoid the conflicts (I am doing this mostly because of the upstream docs is using fabric rather than fabric-ai)

does it make better sense?

Reply:
@chenrui333 The formula name is obvious, but I'm talking about the binary artifact too.

While Homebrew installs it under a different absolute path, if installed along with `fabric`, this will result in 2 different `fabric` binaries in the PATH, which is confusing for people. Also, most Linux distributions and operating systems will either have to rename it to `fabric-ai` like in this PR or introduce conflict spec with the `fabric` package, which doesn't make much sense, since `fabric-ai` can work perfectly as a standalone binary.

After this PR is merged, I'm going to open a PR under fabric repository to update documentation and reference `fabric-ai` if it's installed via Homebrew.

Reply:
okay, cool as long as the upstream documentation updates to use `fabric-ai` rather than `fabric`, I think I am cool with this PR merge. Thanks for the chat!

Reply:
@chenrui333 to close the loop here: the documentation PR has been merged and all info is up to date now: https://github.com/danielmiessler/fabric?tab=readme-ov-file#using-package-managers

Reply:
@vaygr very nice, thanks a lot! ❤️